### DISCUSSION TITLE
Link Pull Request to release?

### DISCUSSION THREAD

**Discussion by rantoniuk** [community/community#23649] [Repositories]
<p>We can link PRs to issues, projects etc.<br>
But I don’t see an easy way to auto-link (or manually) a Pull Request to a particular release on the UI level. I think that would tremendously help to see in which version a PR has been incorporated.<br>
Am I missing something?</p>

**Comment by ernest-phillips** [comment]
<p>Hi <a class="mention" href="/warden">@warden</a> :wave:</p>
<p>Welcome! Your idea is so great that we are already working on it!  I do not have any details for when you might see this in the product, but you can keep an eye on the <a href="https://github.blog/changelog/">GitHub Changelog</a> for updates.</p>

**Reply by ijshilpa** [reply]
@ernest-phillips is there any update on the feature?

**Comment by rantoniuk** [comment]
@ernest-phillips it's been almost two years and I still don't see how to link an issue to a release(d version).
Do you have any updates on this?

**Comment by LudovicTOURMAN** [comment]
Early 2023 and I'm still expecting such a feature to be available :heart:

**Comment by rantoniuk** [comment]
@ernest-phillips since this is still not implemented, could you add respective project label to it? Namely "Product Feedback" so that it is easy to find and people can vote on this.
